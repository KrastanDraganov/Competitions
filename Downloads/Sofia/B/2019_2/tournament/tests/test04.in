50
5 14 31 33 36 40
29 0 3 4 6 10 11 12 14 15 18 20 23 24 27 28 29 31 33 35 36 37 39 40 41 42 43 46 47 49
44 0 1 3 4 5 6 7 8 9 10 11 12 13 14 15 16 18 19 20 21 22 23 24 26 27 28 29 30 31 33 34 35 36 37 38 39 40 41 42 43 46 47 48 49
14 0 6 10 11 12 20 22 23 28 33 36 40 41 42
19 0 3 6 10 11 12 20 23 28 30 31 33 36 37 39 40 41 42 49
36 0 1 3 4 6 7 9 10 11 12 14 15 18 20 21 22 23 24 26 27 28 29 31 33 35 36 37 38 39 40 41 42 43 46 47 49
7 0 19 31 33 36 40 41
33 0 1 3 4 6 9 10 11 12 14 15 18 20 22 23 24 26 27 28 29 31 33 35 36 37 39 40 41 42 43 46 47 49
41 0 1 3 4 5 6 7 9 10 11 12 13 14 15 16 18 19 20 21 22 23 24 26 27 29 31 32 33 34 35 36 37 38 39 40 41 42 43 46 47 49
30 0 1 3 4 6 10 11 12 14 15 18 20 23 24 27 28 29 31 33 35 36 37 39 40 41 42 43 46 47 49
13 0 6 11 12 20 23 28 31 33 36 40 41 42
7 0 6 31 33 36 40 41
11 0 6 11 15 20 23 31 33 36 40 41
38 0 1 3 4 5 6 7 9 10 11 12 14 15 18 20 21 22 23 24 26 27 28 29 31 32 33 35 36 37 38 39 40 41 42 43 46 47 49
23 3 4 6 10 11 12 18 20 23 24 27 28 29 31 33 36 37 39 40 41 42 46 49
23 0 3 4 6 10 11 14 18 20 23 24 27 28 29 31 33 36 39 40 41 42 46 49
38 0 1 3 4 5 6 7 9 10 11 12 13 14 15 18 20 21 22 23 24 26 27 28 29 31 33 35 36 37 38 39 40 41 42 43 46 47 49
48 0 1 2 3 4 5 6 7 8 9 10 11 12 13 14 15 16 18 19 20 21 22 23 24 25 26 27 28 29 30 31 32 33 34 35 36 37 38 39 40 41 42 43 44 46 47 48 49
23 0 3 4 6 10 11 12 20 23 24 27 28 29 31 33 36 37 39 40 41 42 46 49
40 0 1 3 4 5 7 9 10 11 12 13 14 15 16 18 20 21 22 23 24 26 27 28 29 30 31 33 34 35 36 37 38 39 40 41 42 43 46 47 49
8 0 6 11 31 33 36 40 41
36 0 1 3 4 6 7 9 10 11 12 14 15 18 20 22 23 24 26 27 28 29 31 33 35 36 37 38 39 40 41 42 43 45 46 47 49
31 0 1 4 6 9 10 11 12 14 15 18 20 23 24 27 28 29 31 33 34 35 36 37 39 40 41 42 43 46 47 49
9 0 6 11 20 31 33 36 40 41
22 0 3 4 6 10 11 12 20 23 28 29 30 31 33 36 37 39 40 41 42 46 49
46 0 1 2 3 4 5 6 7 8 9 10 11 12 13 14 15 16 18 19 20 21 22 23 24 26 27 28 29 30 31 32 33 34 35 36 37 38 39 40 41 42 43 46 47 48 49
32 0 1 3 4 6 9 10 11 12 14 15 18 20 22 23 24 27 28 29 31 33 35 36 37 39 40 41 42 43 46 47 49
21 0 3 4 6 10 11 12 20 23 24 28 29 31 33 36 39 40 41 42 46 49
13 0 6 8 11 12 20 23 31 33 36 40 41 42
20 0 3 4 6 10 11 12 20 23 28 31 33 35 36 37 39 40 41 42 49
38 0 1 3 5 6 7 8 9 10 11 12 13 14 15 16 18 20 21 22 23 26 27 28 29 31 33 35 36 37 38 39 40 41 42 43 46 47 49
3 3 33 40
41 0 1 2 3 4 5 6 7 9 10 11 12 14 15 16 18 19 20 21 22 23 24 26 27 28 29 30 31 33 34 35 36 37 38 39 40 41 42 43 47 48
1 40
39 0 1 3 4 5 6 7 9 10 11 12 13 14 15 16 18 20 21 23 24 26 27 28 29 30 31 33 35 36 37 38 39 40 41 42 43 46 47 49
25 0 3 4 6 10 11 12 14 15 18 20 23 24 27 28 31 33 36 37 39 40 41 42 46 49
4 31 33 40 41
18 0 3 6 10 11 12 15 20 23 27 28 31 33 36 39 40 41 42
34 0 1 3 4 6 7 9 10 11 12 14 15 18 20 22 23 24 26 27 28 29 31 33 35 36 37 39 40 41 42 43 46 47 49
15 0 3 6 10 11 12 20 23 28 31 33 36 40 41 42
0
4 0 31 33 40
11 0 6 11 12 20 23 31 33 36 40 41
27 0 3 4 6 10 11 12 14 15 18 20 23 24 27 28 29 31 33 35 36 37 39 40 41 42 46 49
47 0 1 2 3 4 5 6 7 8 9 10 11 12 13 14 15 16 18 19 20 21 22 23 24 25 26 27 28 29 30 31 32 33 34 35 36 37 38 39 40 41 42 43 46 47 48 49
48 0 1 2 3 4 5 6 7 8 9 10 11 12 13 14 15 16 17 18 19 20 22 23 24 25 26 27 28 29 30 31 32 33 34 35 36 37 38 39 40 41 42 43 44 46 47 48 49
21 0 3 4 6 10 11 12 20 23 28 29 31 32 33 36 37 39 40 41 42 49
28 0 3 4 6 10 11 12 14 15 18 20 23 24 27 28 29 31 33 35 36 37 39 40 41 42 43 46 49
43 0 1 3 4 5 6 7 8 9 10 11 12 13 14 15 16 18 19 20 21 22 23 24 26 27 28 29 30 31 33 34 35 36 37 38 39 40 41 42 43 46 47 49
18 0 3 6 10 11 12 20 23 28 31 32 33 36 37 39 40 41 42
