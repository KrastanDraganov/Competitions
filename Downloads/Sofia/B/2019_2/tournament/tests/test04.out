45 17 44 25 48 47 46 49 32 2 34 30 38 43 29 35 37 39 28 8 19 16 13 5 21 7 26 22 9 1 15 27 24 4 3 10 42 12 23 11 6 41 0 14 18 20 36 31 33 40 
